torch/utils/cpp_extension.py

```diff
@@ -283,6 +283,25 @@ def _join_sycl_home(*paths) -> str:
     '-D__HIP_NO_HALF_CONVERSIONS__=1',
 ]
 
+COMMON_SYCL_FLAGS = [
+    '-fsycl',
+    '-fsycl-targets=spir64_gen,spir64',
+]
+
+# "preview breaking changes" flags allow to support both C++ ABIs
+COMMON_SYCL_CCFLAGS = [
+    '-fpreview-breaking-changes',
+    '-D__INTEL_PREVIEW_BREAKING_CHANGES',
+]
```

Comment:
We have upgraded the oneAPI to the latest version, the two flags are not necessary.

Reply:
removed